gulpfile.js

```diff
@@ -21,6 +21,7 @@ var gulp = require('gulp'),
     connectModRewrite = require('connect-modrewrite'),
     ngConfig = require('gulp-ng-config'),
     prettyError = require('gulp-prettyerror'),
+    path = require('path'),
```

Comment:
Have you tried to run dev again and again while running watch task as mentioned in issue?